src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -1331,30 +1324,22 @@ private bool IsPartialType()
             return false;
         }
 
-        private bool IsPartialMember()
+        /// <summary>
+        /// Check if this is a contextual keyword followed by a member declaration with a type
+        /// followed by identifier.
+        /// </summary>
+        private bool IsContextualMember()
```

Comment:
>IsContextualMember [](start = 21, length = 18)

"ContextualMember" is really confusing term think we should come up with a better name. IsPossibleTypeFollowedByAMemberName?